Lib/pydoc.py

```diff
@@ -1769,7 +1770,8 @@ def render_doc(thing, title='Python Library Documentation: %s', forceload=0,
               _getdoc(object)):
         # If the passed object is a piece of data or an instance,
         # document its available methods instead of its value.
-        if hasattr(object, '__origin__'):
+        if (hasattr(object, '__origin__') and
+            not isinstance(object, typing._AnnotatedAlias)):
```

Comment:
Please, add a comment here: why do we special case it.

Reply:
done